src/containers/dynamicarray.d

```diff
@@ -150,6 +150,22 @@ struct DynamicArray(T, Allocator = Mallocator, bool supportGC = shouldAddGCRange
 		}
 	}
 
+	/** Deletes all the elements, but does not disallocate the memory
+	of the DynamicArray.
+	*/
+	void clean()
```

Comment:
The other containers in this library as well as the containers in Phobos call this `clear`.